### PR TITLE
feat(ext/pip): pip install and import

### PR DESCRIPTION
**Pull request by eliassjogreen**
Depends on #42

### CODE REVIEW

**Review by DjDeveloperr — APPROVED** [review]
LGTM!

File: ext/pip.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+   *
+   * @param module The Python module which you wish to install
+   *
+   * @example
+   * ```ts
+   * import { python } from "https://deno.land/x/python/mod.ts";
+   * import { install } from "https://deno.land/x/python/ext/pip.ts";
+   *
+   * await install("fuckit");
+   * const fuckit = python.import("fuckit");
```
Comment (lino-levan) [inline ext/pip.ts:73]: 🤨

File: test/deps.ts [hunk 2]
```diff
@@ -1 +1 @@
-export * from "https://deno.land/std@0.178.0/testing/asserts.ts";
+export * from "https://deno.land/std@0.195.0/testing/asserts.ts";
```
Comment (lino-levan) [inline test/deps.ts:1]: already out of date

File: ext/pip.ts (hunk 1 above)
Comment (eliassjogreen) [reply ext/pip.ts]: Oops, had a really frustrating evening to say the least.

**Pull request merged**